charts/tenant-base/chart/templates/bucketclaim.yaml

```
@@ -0,0 +1,10 @@
+{{- range .Values.bucketClaims }}
+---
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: {{ printf "%s-%s" .name "bucket" }}
+spec:
+  generateBucketName: {{ printf "%s-%s" .name "bucket" }}
```

Same, we don't want a name conflict:
```suggestion
  generateBucketName: {{ printf "%s-%s" (include "chart.fullname" $) .name }}
```